index.html

```diff
@@ -592,91 +710,101 @@ <h3>
           following steps:
         </p>
         <ol>
-          <li>Let |axisValues:list| be a [=list=] of {{unsigned long}} values
-          representing the most recent logical axis input values for each axis
-          input of the device represented by |gamepad|.
-          </li>
-          <li>Let |maxRawAxisIndex:long| be the [=list/size=] of |axisValues| −
-          1.
+          <li>Let |rawGamepad| be the [=gamepad=] represented by |gamepad|.
           </li>
-          <li>[=list/For each=] |rawAxisIndex:long| of [=the range=] from 0 to
-          |maxRawAxisIndex|:
+          <li>[=list/For each=] |rawAxisIndex:long| of the [=list/indices=] of
+          the [=axis list=] of |rawGamepad|:
             <ol>
+              <li>Let |rawAxis| be the [=axis=] at index |rawAxisIndex| of the
+              [=axis list=].
```

Comment:
Can we set it just like this?
```suggestion
              <li>Let |rawAxis| be [=axis list=][|rawAxisIndex|].
```

Reply:
I think this doesn't work because [=axis list=] doesn't refer to a specific list, maybe like this:

* Let |axisList| be the [=axis list=] of the [=gamepad=] represented by |gamepad|.
* [=list/For each=] |sourceIndex| of the [=list/indices=] of |axisList|:
  * Let |mappedIndex| be |gamepad|.[[axisMapping]][|sourceIndex|].
  * Set |gamepad|.[[axes]][|mappedIndex|] to the current [=normalized axis value=] for |axisList|[|sourceIndex|].

Reply:
sgtm